manifest.yaml

```diff
@@ -23,18 +23,18 @@ tests:
     type: startup
     group: metadataproxy.dev
 deploy:
-  - name: development
```

Comment:
Also removing the development step as this happens automatically these days as part of the CI checks after merging to master.